model/canvas.cc

```diff
@@ -395,8 +395,20 @@ namespace minsky
   
   ItemPtr Canvas::itemAt(float x, float y)
   {
-    auto item=model->findAny(&Group::items,
-                        [&](const ItemPtr& i){return i->visible() && i->contains(x,y);});
+    // Fix for library dependency problem with items during Travis build     
+    ItemPtr item;                    
+    auto minD=numeric_limits<float>::max();
+    model->recursiveDo(&GroupItems::items,
+                       [&](const Items&, Items::const_iterator i)
+                       {
+                          float d=sqr((*i)->x()-x)+sqr((*i)->y()-y);
+                          if (d<minD && (*i)->visible() && (*i)->contains(x,y))
+                            {
+                              minD=d;
+                              item=*i;
+                            }
+                          return false;  
+                       });
     if (!item)
```

Comment:
This statement is now redundant

Reply:
Thanks, I didn't understand why I still needed it, but on a previous iteration the code seemed to find the item everywhere on the canvas, so I added it back in, but I must have had some other bit of code that caused that. I will remove it and recommit for a pull request.